cmd/ui/src/views/DatabaseManagement/DatabaseManagement.tsx

```diff
@@ -0,0 +1,293 @@
+// Copyright 2024 Specter Ops, Inc.
```

Comment:
I think it might be helpful to break the reducer, hook, and component out into separate files here